src/amo/components/CollectionManager/index.js

```diff
@@ -56,12 +68,17 @@ type State = {|
   description?: string | null,
   name?: string | null,
   slug?: string | null,
+  addAddonStatus: AddonAddedStatusType,
```

Comment:
Shouldn't this be positioned first?

Reply:
Also, is it `status` or `state`?

Reply:
Let's make it `status`. I'll update it elsewhere.